src/main/java/com/notably/logic/commands/EditCommand.java

```diff
@@ -28,7 +28,8 @@ public EditCommand(Body body) {
     public void execute(Model notablyModel) throws CommandException {
         requireNonNull(notablyModel);
         try {
-            notablyModel.updateCurrentlyOpenBlockBody(body);
+            //notablyModel.updateCurrentlyOpenBlockBody(body);
+            notablyModel.setBlockEditable(true);
         } catch (CannotModifyRootException ex) {
             throw new CommandException(ex.getMessage());
         }
```

Comment:
@HemanshuGandhi you can actually edit this directly to

```java
public void execute(Model notablyModel) {
  requireNonNull(notablyModel);
  notablyModel.setBlockEditable(true);
}
```

Reply:
alright! Rectified.